setup/github-bot/README.md

```diff
@@ -0,0 +1,25 @@
+# node.js Build github-bot Setup
+
+For setting up a Debian 8 box for github-bot
+
+```text
+Host infra-rackspace-debian8-x64-2
+  HostName 119.9.53.51
+  User root
+```
+
+Note that these hostnames are also used in the *ansible-inventory* file. The IP addresses will need to be updated each time the servers are reprovisioned.
+
+To set up a host, run:
+
+```text
+$ ansible-playbook -i ../ansible-inventory ansible-playbook.yaml
+```
+
+Start-up of the github-bot service:
+
+```text
+$ systemctl start github-bot
```

Comment:
This will probably already be made by the sensible script; perhaps nitpick but I would suggest how to restart it instead.